test/support/websocket_server/socket_handler.ex

```diff
@@ -0,0 +1,33 @@
+defmodule TestWebSocket.Handler do
+  @behaviour :cowboy_websocket
+
+  @impl true
+  def init(request, state) do
+    case :ets.whereis(:clients) do
+      :undefined -> :ets.new(:clients, [:named_table, :public])
+      _ref -> nil
+    end
+
+    {:cowboy_websocket, request, state}
```

Comment:
```suggestion
    {:cowboy_websocket, request, state}
```
`TestWebSocket.ClientConnection.store/1` seems to be an agent and does not utilize ETS there is no point in initializing ets here.

Reply:
I would expect `TestWebSocket.ClientConnection.start_link` to be called here instead. I may be wrong though. It is started manually in the setup clause so after all, you might just want to delete that ets init thing.

Reply:
Yup, I just forgot to delete this ets fragment.